src/data/feature_index.js

```diff
@@ -17,15 +17,19 @@ import type CollisionIndex from '../symbol/collision_index';
 import type StyleLayer from '../style/style_layer';
 import type {FeatureFilter} from '../style-spec/feature_filter';
 import type {CollisionBoxArray} from './array_types';
+import type Transform from '../geo/transform';
 
 const {FeatureIndexArray} = require('./array_types');
 
 type QueryParameters = {
     scale: number,
     bearing: number,
+    cameraToCenterDistance: number,
```

Comment:
Why do we split `cameraToCenterDistance` out as its own parameter in all these calls? It seems like if we're passing in the transform, we can just get it from the transform when we need it?